platform-sdk/docs/proposals/template-2.md

```diff
@@ -0,0 +1,184 @@
+# Proposal Title Goes Here
+
+| Metadata           | Entities                            | 
+|--------------------|-------------------------------------|
+| Designers          | John, Mary, alpha-team              |
+| Functional Impacts | Services, DevOps, Mirror Node, etc. |
+| Related Proposals  | Proposal-1, Proposal-2              |
+| HIPS               | HIP-1, HIP-2,                       |
+| State              | DRAFT                               |
+
+## Summary
+
+This section should give a short answer to the question: "What is this proposal about?"
+
+Give a 1-3 sentence summary of the capability in the design proposal.
+
+## Purpose and Context
+
+This section should give an answer to the question: "Why is this proposal necessary?"
+
+All subtitled sections are optional and can be removed if not applicable.
+A minimal proposal must have a purpose and context section without any sub-sections.
+In any case the section must give a detailed answer to the question: "Why is this proposal necessary?"
+
+### Motivation
+
+Describe the motivation for the proposal. What problem does it solve? What use cases does it support?
```

Comment:
This seems redundant with the `Purpose and Context` section. The question of "Why is this proposal necessary?" is the the same as "What problem does this solve?"

Reply:
Agree as long as you have text under the `Purpose and Context` header instead of just having it as a wrapper around all the h3 headers. My idea was more like that:

```
## Purpose and Context

### Motivation
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.

### Goals
Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.

### Non-Goals
At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
```

Reply:
I like the goals and non-goals as explicit substructure to purpose and context.   I feel that the motivation section is redundant with the content I would expect to see at the top-level of `Purpose and Context`.